### ISSUE TITLE
Bluray: No Timeline and no pause button

### ISSUE THREAD

**Issue by CS-Alchemist** [xbmc/xbmc#21783]
### Describe the bug
Here is a clear and concise description of what the problem is:

When certain blurays are played, the video player may misbehave.
This affects the comfort level when playing blurays.

I have encountered the following bugs during video playback:

### No information about the playing time.
Correct timeline is not displayed.
The maximum playing time of the movie is not displayed
Only the time is counted
There is no progress bar

### No pause button
It is not possible to pause the current movie in this state.
There is no pause button in the interface.
Pressing the keyboard shortcut for pause does not work.

All blurays are played with menu. The playback works without any problems. Any video can be played through the menu and the menu navigation works fine. The problem is that the interface of the video player does not show all the information and does not provide all the buttons that the playback should have.

## Expected Behavior
Here is a clear and concise description of what was expected to happen:

- Individual video elements of a Bluray disk should be pausable when played with MENU.
- A progress bar should be displayed

### No information about the playing time.
- Correct timeline is not displayed.
- The maximum playing time of the movie is not displayed
- Only the time is counted
- There is no progress bar

### No pause button
- It is not possible to pause the current movie in this state.
- There is no pause button in the interface.
- Pressing the keyboard shortcut for pause does not work.

## Possible Fix

This error only occurs with blurays that have a few second m2ts file, which is used for menu playback. (i.e. as background image)
In the corresponding menu it looks like the m2ts file is paused.
Exactly from this point on the video player does not work properly.
Only the time is counted up. 
The state of the timeline does not change when a movie/episode is started through the menu.

The error can only be avoided if the bluray is not played with menu, but the corresponding playlist is selected in the simplified Kodi menu. This will play the movie directly without any bugs.

### To Reproduce

If you have the blurays of the series Heros, you can test it with it. Because with all Blurays of the series the bug occurs. I suspect that with some Universal products could also occur.
In my case, the German edition of Garden of Sinners also has the problem. 
The cause and effect is the same for both collections.

1.  Play a bluray with Menu: The bluray should have a short .m2ts file that serves as a background image for a menu.
2. Wait until the menu appears and select a entry from the menu.
3.  Check timeline and buttons

### Debuglog

The debuglog can be found here:

[Garden_of_Sinners.log](https://github.com/xbmc/xbmc/files/9435298/Garden_of_Sinners.log)
[Heros.log](https://github.com/xbmc/xbmc/files/9435299/Heros.log)

### Screenshots 
Here are some links or screenshots to help explain the problem:

### Faulty interface
![GoS_BUG](https://user-images.githubusercontent.com/95227569/186963234-f961bbfa-094e-4ffc-8aeb-11134804a948.png)
![HEROS_BUG](https://user-images.githubusercontent.com/95227569/186963246-481d1467-65f8-4aca-a373-4831c8e82a74.png)
### Correct interface
![GoS_WA](https://user-images.githubusercontent.com/95227569/186963244-009ca48a-5f70-4ac4-a1cb-8785892ee7a3.png)
![HEROS_WA](https://user-images.githubusercontent.com/95227569/186963250-e9bb8394-2183-45de-857a-e954bde401d3.png)

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

### Your Environment
Used Operating system:

 - [x] Android
 - [ ] iOS
 - [ ] tvOS
 - [x] Linux
 - [ ] OSX
 - [x] Windows
 - [ ] Windows UWP

 - Operating system version/name:
 - Kodi version: 19.4 and 19.90.705

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by enen92** [comment]
Are you able to give it a go on a recent nightly to see if the issue persists?

**Comment by CS-Alchemist** [comment]
I just tried it again with the latest nightly for android.
The error is still present

**Comment by enen92** [comment]
Thanks. I'd like to help but I am not sure how I'll be able to so without an actual sample.

**Comment by CS-Alchemist** [comment]
There is a Sample Online:
https://anonfiles.com/N8f7485ay2/Heros.tar_gz

use 00009.mpls/00008.m2ts/menu->Extras->Making-Of as Referance file

**Comment by enen92** [comment]
Thanks, I have downloaded the sample and will look into it at some point next week

**Comment by enen92** [comment]
@CS-Alchemist the issue should be fixed with the patch above. Testbuilds:

win-64: https://mirrors.kodi.tv/test-builds/windows/win64/KodiSetup-20220904-d8b9f2c7-timeline-x64.exe
android (arm): https://mirrors.kodi.tv/test-builds/android/arm/kodi-20220904-d8b9f2c7-timeline-26440-armeabi-v7a.apk

Mind give them a go with your files to check if the issue persists?

Thanks

**Comment by ShubuYeolekar** [comment]
Do u use external blu ray player to play discs or iso? are you able to see blu ray with menu ?

**Comment by CS-Alchemist** [comment]
@enen92 
Your commit seems to work pretty well. I have tested it with all my Blu-rays that caused this error. For each episode/movie on each disc, the timeline is now displayed as intended and pausing is also possible.

I suspected that the state machine of the player could not change its state in this sittuation. But I was not able to find the place in the code.

Thank you very much for the effort.

Unfortunately I could not run the Windows binary, because the gui would not start. I think I have missing dependency for that on my windowsmachine. But with the Android apk I could test everything for now.
